phoenix-core/src/main/java/org/apache/phoenix/execute/MutationState.java

```diff
@@ -1239,15 +1239,17 @@ private void filterIndexCheckerMutations(Map<TableInfo, List<Mutation>> mutation
                     } else if (m instanceof Put) {
                         long timestamp = IndexRegionObserver.getMaxTimestamp(m);
 
-                        // Phase 1 index mutations are set to unverified
-                        ((Put) m).addColumn(emptyCF, emptyCQ, timestamp, IndexRegionObserver.UNVERIFIED_BYTES);
-                        addToMap(unverifiedIndexMutations, tableInfo, m);
+                        // Phase 1 index mutations are set to unverified.
+                        // Just send empty with Unverified
+                        Put unverifiedPut = new Put(m.getRow());
+                        unverifiedPut.addColumn(emptyCF, emptyCQ, timestamp, IndexRegionObserver.UNVERIFIED_BYTES);
+                        addToMap(unverifiedIndexMutations, tableInfo, unverifiedPut);
 
                         // Phase 3 mutations are verified
-                        Put verifiedPut = new Put(m.getRow());
-                        verifiedPut.addColumn(emptyCF, emptyCQ, timestamp,
+                        // Send entire mutation with verified
+                        ((Put) m).addColumn(emptyCF, emptyCQ, timestamp,
```

Comment:
The empty column is already part of the mutation at this moment with value "x". Can we remove it first and add the new one as the patch for PHOENIX-5539 does.